src/app/_components/Event/SkeletonCard.js

```diff
@@ -0,0 +1,10 @@
+const fakeCardArray = [7, 8, 9, 10, 11, 12, 13, 14];
```

Comment:
@Yivgha there is no need for separate array with this numbers. `new Array` is more idiomatic for iterating

Reply:
same as above